lte/gateway/docker/README.md

```diff
@@ -47,6 +47,13 @@ docker-compose build
 docker-compose up
 ```
 
+Optional: If you want to connect to an orc8r, copy the `rootCA.pem` from the orc8r
```

Comment:
I think this should happen before we run `docker-compose up` for the first time, so we should move this above.

Reply:
Since we mount the directory anyway I would have assumed that the order doesn't really matter, but I can change it if that is better

Reply:
I guess the AGW will read the certificate on startup and probably not re-read it when it is copied later when it is already started.

Reply:
I have moved it before the docker setup.